src/minishell.c

```diff
@@ -0,0 +1,59 @@
+#include <assert.h>
+#include <stdlib.h>
+
+#include "debugger.h"
+#include "libft.h"
+
+#include "minishell.h"
+#include "shellcore.h"
+
+extern char **environ;
```

Comment:
Do we need environ variable knowing that we already have the variable with "char *envp[]" ?

Reply:
that's for using a direct call to shell_exec(), see line 53